gems/sorbet-runtime/lib/types/private/methods/_methods.rb

```diff
@@ -122,15 +132,55 @@ def self.signature_for_method(method)
     @signatures_by_method[key]
   end
 
+  # a module A such that:
+  #   if A is included or extended by a module B,
+  #   then we want B to have the hooks installed on itself
+  # should extend this module.
+  module VirallyInstallHooks
```

Comment:
I thought I added this as a comment on the lat pull request but I think the comment got deleted somehow. The comment was:

The word "viral" has negative connotations (infection, sickness) and I don't want it showing up in people's backtraces. Can we change this to a less negative word? Like "transitive" or something else.